src/connect_dialog_handler.rs

```diff
@@ -168,81 +170,114 @@ impl ConnectDialog {
             border_radius: 2.0,
         };
 
+        let text_box_container_style = ContainerStyle {
+            border_color: Color::WHITE,
+            background_color: Color::WHITE,
+            border_width: 0.0,
+            border_radius: 0.0,
```

Comment:
Maybe play with some subtle borders for the text box, couple of pixel line, not too bright, some corner radius?

We should establish the style for such "information" boxes and then use the same whenever we add one